prov/sm2/src/sm2_fifo.h

```diff
@@ -241,4 +241,30 @@ static inline void sm2_fifo_write_back(struct sm2_ep *ep,
 	sm2_fifo_write(ep, xfer_entry->hdr.sender_gid, xfer_entry);
 }
 
+static inline const char *sm2_print_ptr(struct sm2_ep *ep, void *ptr)
+{
+	static char outstr[80];
+	struct sm2_av *av;
+	struct sm2_mmap *map;
+	av = container_of(ep->util_ep.av, struct sm2_av, util_av);
+	map = &av->mmap;
+	struct sm2_coord_file_header *header = (void *) map->base;
```

Comment:
Fix variable declaration block - declare (inline initialization is ok), space, rest of code.
Also I think Seth added a helper function to simplify the map pointer get